libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -478,10 +478,6 @@ MQTTStatus_t MQTT_GetPingreqPacketSize( size_t * pPacketSize );
  */
 MQTTStatus_t MQTT_SerializePingreq( const MQTTFixedBuffer_t * const pBuffer );
 
-MQTTStatus_t MQTT_GetIncomingPacket( MQTTTransportRecvFunc_t recvFunc,
```

Comment:
Why is this removed. I think we are planning to implement this function.

Reply:
No, it's not in our API doc, and we wrote MQTT_GetIncomingPacketTypeAndLength instead. I believe we intended to remove it before, but never got around to it.

Reply:
I think the plan is to still keep it and implement it. It is an additional API to provide the customer the choice of avoiding multiple transport reads. It came up in the design proposal.

Reply:
After some offline discussion it does seem like that is the plan, so I will re add this function.